src/secure-value.ts

```diff
@@ -340,7 +367,7 @@ export const serializedSecureEnvSourceText = (function secureEnvFactory(rawEnv:
             try {
                 const rawThisArg = rawEnv.getRawValue(thisArg);
                 const rawArgArray = rawEnv.getRawValue(argArray);
-                raw = apply(rawTarget as RawFunction, rawThisArg, rawArgArray);
+                raw = rawApplyHook(rawTarget as RawFunction, rawThisArg, rawArgArray);
```

Comment:
marking it with the `raw` prefix is important so we know we need try/catch protection to avoid stack overflows.